templates/workspace_services/azureml/template_schema.json

```diff
@@ -3,19 +3,59 @@
   "$id": "https://github.com/microsoft/AzureTRE/templates/workspace_services/azureml/template_schema.json",
   "type": "object",
   "title": "Azure Machine Learning",
-  "description": "Installs Azure Machine Learning. Please be aware this template opens up additional firewall rules to enable Azure ML to function.",
+  "description": "Azure Machine Learning",
   "required": [],
   "properties": {
+    "display_name": {
+      "type": "string",
+      "title": "Name for the workspace service",
+      "description": "The name of the workspace service to be displayed to users",
+      "default": "Azure Machine Learning",
+      "updateable": true
+    },
+    "description": {
+      "type": "string",
+      "title": "Description of the workspace service",
+      "description": "Description of the workspace service",
+      "default": "Azure Machine Learning empowers data scientists and developers to build, deploy, and manage high-quality models faster and with confidence.",
+      "updateable": true
+    },
+    "overview": {
+      "type": "string",
+      "title": "Workspace Service Overview",
+      "description": "Long form description of the workspace service, in markdown syntax",
+      "default": "",
+      "updateable": true
+    },
```

Comment:
Isn't that the generic text anyway? Why should we define it in a specific template definition?

Reply:
I've added a new default.